Corona-Warn-App/src/main/assets/privacy_en.html

```diff
@@ -25,28 +128,25 @@
     <strong>6. Why is your data processed?</strong>
 </p>
 <p>
-    <strong>7. How does the transnational warning system work?</strong>
+    <strong>7. What permissions does the app require?</strong>
 </p>
 <p>
-    <strong>8. What permissions does the app require?</strong>
+    <strong>8. When will your data be deleted?</strong>
 </p>
 <p>
-    <strong>9. When will your data be deleted?</strong>
+    <strong>9. Who will receive your data?</strong>
 </p>
 <p>
-    <strong>10. Who will receive your data?</strong>
+    <strong>10. Is your data transferred to countries outside the EU?</strong>
 </p>
 <p>
-    <strong>11. Is your data transferred to countries outside the EU?</strong>
+    <strong>11. How can you withdraw your consent?</strong>
 </p>
 <p>
-    <strong>12. How can you withdraw your consent?</strong>
+    <strong>12. What other rights do you have under data protection law?</strong>
 </p>
 <p>
-    <strong>13. What other rights do you have under data protection law?</strong>
-</p>

```

Comment:
the document contains of 14 chapters and the chapter 13 is 
13. What other rights do you have under data protection law?

Reply:
all correct i was wrong